packages/transaction-messages/src/compile/message.ts

```diff
@@ -46,6 +49,11 @@ type VersionedCompiledTransactionMessage = BaseCompiledTransactionMessage &
         version: number;
     }>;
 
+const EMPTY_BLOCKHASH_LIFETIME_CONSTRAINT = {
+    blockhash: '11111111111111111111111111111111',
+    lastValidBlockHeight: 0n,
```

Comment:
I know that `getCompiledLifetimeToken()` ignores this value so it can be literally anything, but this is maybe confusing to the reader (ie. that it's here).

We could obviate the need for this by doing…

```ts
const LIFETIME_TOKEN_WHEN_LIFETIME_ABSENT = '11111111111111111111111111111111';

/* ... */
    lifetimeToken: transactionMessage.lifetimeConstraint
        ? getCompiledLifetimeToken(transactionMessage.lifetimeConstraint)
        : LIFETIME_TOKEN_WHEN_LIFETIME_ABSENT,
```